scripts/apps/workspace/content/services/ContentService.ts

```diff
@@ -344,21 +347,37 @@ export function ContentService(api, templates, desks, packages, archiveService,
      * Handle drop event transfer data and convert it to an item
      */
     this.dropItem = (item: IArticle, {fetchExternal} = {fetchExternal: true}) => {
-        if (item._type !== 'externalsource') {
-            if (item._type === 'ingest') {
-                return send.one(item);
-            }
+        const invalidFiles = [];
+
+        if (appConfig.pictures != null
+            && (item.renditions.original.width < appConfig.pictures.minWidth
```

Comment:
You should probably be using optional chaining when accessing renditions since an item might not have them and an exception would be thrown when you try to access a property of `undefined`